parlai/scripts/interactive_web.py

```diff
@@ -0,0 +1,260 @@
+                      <div class="field is-grouped">
+                        <p class="control is-expanded">
+                          <input class="input" type="text" id="userIn" placeholder="Type in a message">
+                        </p>
+                        <p class="control">
+                          <button id="respond" type="submit" class="button has-text-white-ter has-background-grey-dark">
+                            Submit
+                          </button>
+                        </p>
+                      </div>
+                  </form>
+                </div>
+              </section>
+            </div>
+        </div>
+
+        <script>
+            document.getElementById("interact").addEventListener("submit", function(event){{
+                event.preventDefault()
+                var text = document.getElementById("userIn").value;
+                document.getElementById('userIn').value = "";
+
+                fetch('/interact', {{
+                    headers: {{
+                        'Content-Type': 'application/json'
+                    }},
+                    method: 'POST',
+                    body: text
+                }}).then(response=>response.json()).then(data=>{{
+                    var parDiv = document.getElementById("parent");
+
+                    var article = document.createElement("article");
+                    article.className = "media"
+
+                    var figure = document.createElement("figure");
+                    figure.className = "media-left";
+
+                    var span = document.createElement("span");
+                    span.className = "icon is-large";
+
+                    var icon = document.createElement("i");
+                    icon.className = "fas fa-user fas fa-2x";
+
+                    var media = document.createElement("div");
+                    media.className = "media-content";
+
+                    var content = document.createElement("div");
+                    content.className = "content";
+
+                    var para = document.createElement("p");
+                    var paraText = document.createTextNode(text);
+
+                    var strong = document.createElement("strong");
+                    strong.innerHTML = "You";
+                    var br = document.createElement("br");
+
+                    para.appendChild(strong);
+                    para.appendChild(br);
+                    para.appendChild(paraText);
+                    console.log("paraText")
+                    console.log(paraText);
+                    content.appendChild(para);
+                    media.appendChild(content);
+
+                    span.appendChild(icon);
+                    figure.appendChild(span);
+
+                    article.appendChild(figure);
+                    article.appendChild(media);
+
+                    parDiv.append(article);
+
+                    // Change info for Model response
```

Comment:
Is there a way to do this by reusing the elements created for the Person response as opposed to creating new elements for the model response? I tried reusing, but couldn't get it to work. @pringshia

Reply:
Pretty sure what you're describing should be feasible. Let's debug offline